app/scripts-browserify/map.js

```diff
@@ -1,82 +1,151 @@
 var Map = {
-  init: function (items) {
-    let map;
-
-    const copenhagen = {
-      lat: 55.6761,
-      lng: 12.5683
-    };
-
-    map = new google.maps.Map(document.getElementById('map'), {
-      center: copenhagen,
-      zoom: 12,
-      gestureHandling: 'greedy',
-    });
-
+  // Start the map off over Copenhagen.
+  center: {
+    lat: 55.6761,
+    lng: 12.5683
+  },
+  zoomLevel: 12,
+
+  // Reference to the google.maps.Map instance created in init().
+  googleMap: false,
+
+  // Whether the map has been initialized and has stopped sending events that
+  // indicates the user is interacting with it.
+  mapIsIdle: false,
+
+  // Reference to current set of markers.
+  clusteredMarkers: undefined,
+
+  /**
+   * Returns an Elastic Search {top_left, bottom_right} bounds object.
+   */
+  getEsBounds: function() {
+    console.log("Got es bounds");
```

Comment:
Leftover from debugging or is there a reason for this? ;)

Reply:
debugging :)